test/gui/shared/scripts/helpers/api/webdav_helper.py

```diff
@@ -0,0 +1,14 @@
+import helpers.api.HttpHelper as request
+import helpers.api.Provisioning as provisioning
+from helpers.ConfigHelper import get_config
+from os import path
+
+
+def get_webdav_url():
+    return path.join(get_config('localBackendUrl'), "remote.php/dav", 'files')
+
+
+def file_exists(user, resource_name):
+    url = provisioning.format_json(path.join(get_webdav_url(), user, resource_name))
```

Comment:
not necessary if we care about the status code only
```suggestion
    url = path.join(get_webdav_url(), user, resource_name)
```